src/types/inp.ts

```diff
@@ -37,6 +37,34 @@ export interface INPMetric extends Metric {
   entries: PerformanceEventTiming[];
 }
 
+export type INPSubpart =
+  | 'input-delay'
+  | 'processing-duration'
+  | 'presentation-delay';
+
+/**
+ * Summary information about the slowest script intersecting the INP duration
```

Comment:
I'm still seeing "slowest" script used here rather than longest. Given that I think we're happy with what's in the README, can you update this now as well?

Reply:
Done!